### PR TITLE
squirrel.DebugSqlizer: debug based on PlaceholderHolder (if present)

### PR DESCRIPTION
**Pull request by thmsweaver**
References: https://github.com/Masterminds/squirrel/issues/116

In DebugSqlizer lets check if the provided `Sqlizer` has a `GetPlaceholder` method. If so, retrieve the placeholder, otherwise default to the existing behavior of debugging against `?`

### CODE REVIEW

File: placeholder.go [hunk 1]
```diff
@@ -12,40 +12,63 @@ import (
...
 
 	// Colon is a PlaceholderFormat instance that replaces placeholders with
 	// colon-prefixed positional placeholders (e.g. :1, :2, :3).
-	Colon = colonFormat{}
+	Colon = colonFormat{":"}
 )
 
-type questionFormat struct{}
+type questionFormat struct {
+	value string
```
Comment (lann) [inline placeholder.go]: I don't think these new fields are necessary. You can instead stick the constants into the new methods:
```go
func (questionFormat) debugPlaceholder() string {
    return "?"
}
```

Comment (thmsweaver) [reply placeholder.go]: just curious, does your preference to return the constants stem from a functional vs. OO perspective?

Comment (lann) [reply placeholder.go]: It stems from a "less code is better" perspective. :smile:

**Comment by lann** [comment]
Thanks!

**Pull request merged**